### ISSUE TITLE
Add Activation columns

### ISSUE THREAD

**Issue by mgarzelli** [sverweij/mscgen_js#253]
A lot of other msc generator allow for the user to add activation columns,
these are like vertical boxes over a line that describes that that action is not finished and that perhaps that entity (related to that vertical line) is still processing and waiting.

for more info, please see this 
http://msc-generator.sourceforge.net/help/5.4/Commands.html

there is a section on activate/deactivate.

btw, your tool totally rocks!!!!

**Comment by sverweij** [comment]
hi @mgarzelli  - thanks for the suggestion. I gather you mean something like this?
```
msc {
  wordwraparcs=true;

  a, b, log;

  a =>> b  [activate=true, label="hello"]; // or maybe simply 'activate'
  b -> log [label="a says hello."];
  b -> log [label="I'm saying hello to b back"];
  b >> a   [deactivate=true, label="hello back"];
}
```
![mscgenjs_chart-2](https://user-images.githubusercontent.com/4822597/41681190-248cf152-74d4-11e8-9c75-7bada29fc370.png)

I've played with the idea to add _auto-activation_ for some time. I chickened out, however because the effort-to-benefit ratio to do that was _way_ out of whack (caused by a combination conceptual questions I couldn't resolve satisfactorily and the way mscgenjs is currently implemented).

Manual activation (your feature request) will be somewhat simpler to implement, but not trivial either (Find a friendly syntax. Decide how to handle unbalanced activates/ deactivates. Find a clean way to render it without causing regression). I'll consider it, but give me some nights to sleep on it...

=> What's the concrete use case you have for this feature?
